plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/ItemSerializerHack.java

```diff
@@ -0,0 +1,46 @@
+package com.programmerdan.minecraft.simpleadminhacks.hacks.basic;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHack;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHackConfig;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+public class ItemSerializerHack extends BasicHack implements CommandExecutor {
+
+	public ItemSerializerHack(SimpleAdminHacks plugin, BasicHackConfig config) {
```

Comment:
If this replaces ItemMetaConverterHack, that class should be deleted as well.

Reply:
It doesn't, the intent is to have something to see what an item should look like in a config. ItemMetaConverterHack may not be necessary anymore though, but that'd want some testing to figure out.

Reply:
You've removed ItemMetaConverterHack from the default config though?

Reply:
Whoops, will fix